provider/src/main/kotlin/au/com/dius/pact/provider/ProviderVerifier.kt

```diff
@@ -1451,6 +1451,7 @@ open class ProviderVerifier @JvmOverloads constructor (
           m.invoke(instance)
         }
       } catch (e: Throwable) {
+        logger.warn(e) { "Failed to invoke provider method '${m.name}'" }
```

Comment:
This may seem unnessecary, but the cause/stacktrace of the original throwable is not being logged otherwise